PhysicsTools/TagAndProbe/plugins/ColinsSoperVariablesComputer.cc

```diff
@@ -22,15 +22,15 @@
 #include "PhysicsTools/TagAndProbe/interface/ColinsSoperVariables.h"
 #include "TLorentzVector.h"
 
-class ColinsSoperVariablesComputer : public edm::EDProducer {
+class ColinsSoperVariablesComputer : public edm::global::EDProducer<> {
 public:
   explicit ColinsSoperVariablesComputer(const edm::ParameterSet& iConfig);
   ~ColinsSoperVariablesComputer() override;
 
-  void produce(edm::Event& iEvent, const edm::EventSetup& iSetup) override;
+  void produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSetup) const override;
 
 private:
-  edm::EDGetTokenT<edm::View<reco::Candidate>> parentBosonToken_;
+  const edm::EDGetTokenT<edm::View<reco::Candidate>> parentBosonToken_;
```

Comment:
Since you are cleaning this file, maybe you can also get rid of the useless line 77 (dead initialization spotted by SA)